src/backbone-metal.js

```diff
@@ -451,7 +460,10 @@ deprecate._format = function(prev, next) {
  * @memberOf deprecate
  * @param {*...} - The values to warn in the console.
  */
-deprecate._warn = typeof console !== 'undefined' && (console.warn || console.log) || _.noop;
+deprecate._warn = function() {
+  var warn = _console.warn || _console.log || _.noop;
```

Comment:
Well so should it really just be:
```js
var _console = typeof console !== 'undefined' ? console : {};
deprecate._warn = _console.warn || _console.log || _.noop;
```
or
```js
var _console = typeof console !== 'undefined' ? console : {};
var _warn = _console.warn || _console.log || _.noop;

deprecate._warn = _warn;
```

Or am I missing something?  thoughts?